src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx

```diff
@@ -1,62 +1,100 @@
 import React from 'react';
+import environment from 'platform/utilities/environment';
 
-const VetTecApplicationProcess = () => (
-  <div
-    className={
-      'columns vads-u-margin-top--neg1p5 vads-u-margin-x--neg1p5 medium-screen:vads-l-col--10'
-    }
-  >
-    <h3 className="vads-u-font-size--h4">
-      Enrolling in VET TEC is a two-step process:
-    </h3>
-    <p>
-      First, you’ll need to apply for Veteran Employment Through Technology
-      Education Courses (VET TEC). You’ll get a Certificate of Eligibility (COE)
-      in the mail if we approve your application.
-    </p>
-    <p>
-      <a
-        href={
-          '/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/introduction'
+class VetTecApplicationProcess extends React.Component {
+  providersWebsiteLink = () => {
+    const programs = this.props.institution.programs;
+    return programs[0].providerWebsite === null ? (
+      <p>
+        To learn more about available programs, visit the training provider's
+        website.
+      </p>
+    ) : (
+      <p>
+        To learn more about available programs,{' '}
+        <a
+          href={` https://${programs[0].providerWebsite}`}
```

Comment:
Extra space after the tic

Reply:
I need that space or the linked text and the previous word will be combined.

Reply:
Nevermind, I was able to make it work without that space.